ystia/yorc
# Bugfix/gh 215 k8s service ip missing runtime view

# Pull Request description

## Description of the change
Make ip visible in runtime view of alien4cloud for services deployed on a kubernetes hosted by GKE

### What I did
I made call to kubernetes API to get external IP of nodes

### How I did it
I just added 2 functions that do the job in "utils.go" file

### How to verify it
Deploy any app on GKE that have a service, for example simple appache, and check in alien runtime view that IP is present for attribute "k8s_service_url" 

### Description for the changelog

## Applicable Issues
Bug https://github.com/ystia/yorc/issues/215
